### ISSUE TITLE
Is there a way to turn off the rendering of "!=" as "≠"

### ISSUE THREAD

**Issue by boynton** [zed-industries/community#1123]
### Check for existing issues

- [X] Completed

### Describe the feature

Currently the "!=" sequence in a source file gets rendered as "≠" in a buffer, as well as in the search dialog, etc. I would like to turn off that "feature". Noticed it in both C and Go source files. Is it just not documented, or is this actually hardwired behavior? Would prefer WYSIWYG.

### If applicable, add mockups / screenshots to help present your vision of the feature

<img width="382" alt="Screenshot 2023-03-15 at 1 27 11 PM" src="https://user-images.githubusercontent.com/372157/225435567-a7a958bf-117a-47ea-bdf4-ceedcf61d200.png">

**Comment by jansol** [comment]
It is a feature in some fonts called 'ligatures'.

**Comment by boynton** [comment]
Right, thanks for confirming that. It is cool it can render them, but what I'm asking: is there a way to turn ligature rendering off? Most apps that I use involving monospaced fonts either default to not rendering them, or have a preference to turn it on or off. Defaulting to on is surprising.

**Comment by JosephTLyons** [comment]
We don't have a way to turn them off as of yet - you'll just need to swap to another font that doesn't have them in the meanwhile.  I'm currently using `Jetbrains Mono NL` in order to avoid the same thing.

**Comment by boynton** [comment]
Thanks, looks like Source Code Pro also works for this purpose. Just need the Find and other UI elements to be font-configurable to match [i.e. this](https://github.com/zed-industries/community/issues/1158).

**Comment by iamnbutler** [comment]
Assuming this merges today this should land in around 2 weeks. Thanks @as-cii for grinding on this 🙏🏽

Not only will this allow you to disable ligatures, but it will also let you set other open type features. @rsms has a create playground for open type features here for inter: https://rsms.me/inter/lab/

https://user-images.githubusercontent.com/1714999/225958457-0d8f981f-38e8-4e74-9444-7660cdbd42e1.mp4

**Comment by macarie** [comment]
@iamnbutler, is this new setting something like VSCode's `editor.fontVariations`, but instead of being string-based is object-based?

I'd like to use https://github.com/evilmartians/mono with Zed, but I need a way to set `wdth` and `wght` to make it usable, in addition to line-height (which is tracked here, I guess: https://github.com/zed-industries/community/issues/304).

**Comment by JosephTLyons** [comment]
A setting for this will be landing in v0.79.0.

**Comment by JosephTLyons** [comment]
The setting should now be released. To disable ligatures, use this setting:

```json

{
  "buffer_font_features": {
    "calt": false
  }
}
```

https://docs.zed.dev/configuration/configuring-zed#font-features

**Comment by macarie** [comment]
@JosephTLyons, this feature doesn't seem to work with `cvXY` character variants.

Is this a bug or something that's just not supposed to work at all?

Here's a side-by-side comparison with VSCode (notice the `%%%` in the comment at the bottom):

![image](https://user-images.githubusercontent.com/48020855/229277796-985961d6-c3c9-47a6-afd8-abd32dc342df.png)
![image](https://user-images.githubusercontent.com/48020855/229277819-c23e8dfd-bd5d-46d1-b1b0-bf1efe72e98c.png)
![image](https://user-images.githubusercontent.com/48020855/229277850-4ec6f66c-25e6-4e3c-a6d5-71922527edf5.png)

I've also tried with Inter (notice the `444` in the comment at the bottom):

![image](https://user-images.githubusercontent.com/48020855/229277980-88085617-135a-49a4-819e-404092830b1c.png)

But other features work, like `zero` or `ss01` (which enables the `cv02` variant, the one for the `444` in the screenshot above):

![image](https://user-images.githubusercontent.com/48020855/229278186-09ccd68e-0694-491f-9f45-162d98f6a0ad.png)
![image](https://user-images.githubusercontent.com/48020855/229278251-2d80a616-105d-4db3-9d36-8bea5c82a894.png)
